src/custom/components/swap/TradeSummary/TradeSummaryMod.tsx

```diff
@@ -50,6 +51,11 @@ export default function TradeSummary({
   const isExactIn = trade.tradeType === TradeType.EXACT_INPUT
   const slippageAdjustedAmounts = computeSlippageAdjustedAmounts(trade, allowedSlippage)
 
+  const [slippageOut, slippageIn] = useMemo(
+    () => [slippageAdjustedAmounts[Field.OUTPUT], slippageAdjustedAmounts[Field.INPUT]],
+    [slippageAdjustedAmounts]
+  )
+
```

Comment:
These changes are not related afait, probably u used the wrong base branch?

Reply:
no, it was syntactic sugar for making the change a bit easier, before it was using the entire `slippageAdjustedAmounts[Field.OUTPUT]` name and it got annoying. It's not a major change but I can change it back if we don't like it @anxolin @alfetopito

Reply:
I'm good, I like the change

Reply:
👍

Reply:
okok 👌, I thought it was comming from other PR. NVM